main/solution/post-deployment/config/infra/registrationWAF.yml

```diff
@@ -20,19 +20,50 @@ Resources:
             RateBasedStatement:
               AggregateKeyType: IP
               Limit: ${self:custom.registrationWAF.rateLimit, 500}
+              ScopeDownStatement: # Scope rate limiting to 'api' requests
+                # Documentation: https://docs.aws.amazon.com/waf/latest/developerguide/waf-bot-control-example-scope-down-login.html
+                ByteMatchStatement:
+                  SearchString: "api"
```

Comment:
can we use /api instead of api

Reply:
Deployed and tested that this works and correctly blocks only /api requests. Thanks!